gui/styles/_wizard.styl

```diff
@@ -8,142 +8,89 @@
 
 .on-step-address
   .step
-    transform translate(-100%, 0)
+    transform translate(-100vw, 0)
 
 .on-step-folder
   .step
-    transform translate(-200%, 0)
+    transform translate(-200vw, 0)
 
 .step
   position absolute
   left 0
   top 0
   display flex
-  flex-direction column
+  //flex-direction column
```

Comment:
Ok, we found and understood the sliding code when we starting pairing on this, but I was not sure why the direction was column and not row.